File: js/src/utils/logger.ts
import * as colors from 'colors'

export class Logger {
    public static success(message: string) {
        console.log(colors.green(`[✓] ${message}`));
    }

    public static warn(message: string) {
        console.log(colors.yellow(`[!] ${message}`));
    }

    public static info(message: string) {
        console.log(colors.yellow(`[+] ${message}`));
    }

    public static error(message: string) {
        console.log(colors.red(`[x] ${message}`));
    }
}

File: python/utils/logger.py
from colorama import Fore, Style


class Logger:
    @staticmethod
    def success(message: str):
        print(Fore.GREEN + "[✓] " + message + Style.RESET_ALL)

    @staticmethod
    def warn(message: str):
        print(Fore.YELLOW + "[!] " + message + Style.RESET_ALL)

    @staticmethod
    def info(message: str):
        print(Fore.CYAN + "[i] " + message + Style.RESET_ALL)

    @staticmethod
    def err(message: str):
        print(Fore.RED + "[x] " + message + Style.RESET_ALL)
